packages/shared/src/lib/auxiliary/deep-link/utils/getRawAmountFromSearchParam.ts

```diff
@@ -3,7 +3,7 @@ import { AmountNotAnIntegerError, SendTransactionParameter } from '@auxiliary/de
 export function getRawAmountFromSearchParam(
     searchParams: URLSearchParams,
     parameterKey: SendTransactionParameter
-): string {
+): bigint {
     let rawAmount = searchParams.get(parameterKey)
     const amount = Number(rawAmount)
     if (!Number.isInteger(amount)) {
```

Comment:
number might be greater than max safe int? Better to do a regular expression here for an int?

Reply:
Yup, that makes sense